app/src/main/java/org/fossasia/openevent/general/search/SearchFragment.kt

```diff
@@ -136,8 +138,14 @@ class SearchFragment : Fragment() {
         }
         searchView.setOnQueryTextListener(queryListener)
         rootView.fabSearch.setOnClickListener {
+            showNoInternetScreen(isNetworkConnected())
```

Comment:
move logic to view model, see how its done in login fragment and login fragment vm

Reply:
> move logic to view model, see how its done in login fragment and login fragment vm

But in event fragment it in not done in view model for check network connection, so I did the same. Shold I change it?